src/kibana/directives/aria_expanded.js

```diff
@@ -0,0 +1,23 @@
+define(function (require) {
+  require('modules')
+    .get('kibana')
+    .directive('ariaExpanded', function () {
+      return {
+        restrict: 'A',
+        link: function ($scope, $elem, attrs) {
+          $elem.bind('click', function () {
+            if ($elem[0].getAttribute('aria-expanded') === 'true') {
```

Comment:
would you prefer that I use `.attr()` over the standard javascript function?

Also trying to figure out how to get this to trigger correctly for your other possible trigger. This currently only works if they click the time button in the top right not if there are no results and the time trigger is shown on the main section of the page.

Do you have any ideas for how I might get the directive to accomplish that? That was part of my reasoning originally behind having the check in the actual timepicker functionality so it didn't matter how it was triggered.